xdem/coreg/base.py

```diff
@@ -1316,13 +1359,55 @@ def copy(self: CoregType) -> CoregType:
 
         return new_coreg
 
+    def _parse_bias_vars(self, step: int, bias_vars: dict[str, NDArrayf] | None) -> dict[str, NDArrayf]:
```

Comment:
The non-public `CoregPipeline` function that makes the pipeline work smoothly for both `fit` and `apply`